### ISSUE TITLE
Does reclaiming F3 & F4 work in Sierra

### ISSUE THREAD

**Issue by nicklear** [tekezo/Karabiner-Elements#62]
Not sure if this working yet or not in Elements, but I was enjoying the feature in Karibiner which reclaims F3 & F4 to be function keys instead of Mission Control and Dashboard. If it does work, can you help me with the code to go in the config text file?

**Comment by hirakujira** [comment]
```
"simple_modifications": {
        "vk_mission_control" : "f3",
        "vk_launchpad" : "f4"
}
```

**Comment by nicklear** [comment]
That's not working for me. 

In case it helps, I'm on the latest version 0.90.34. In event viewer I get nothing when I press F3 or F4, even though MC and D are activated. All the other function keys show in event viewer as well as working e.g. brightness down is SysKeyDown 0x3 then SysKeyUp 0x3.

**Comment by tekezo** [comment]
You can do it by this json.

```json
{
    "profiles": [
        {
            "name": "Default profile",
            "selected": true,
            "simple_modifications": {
            },
            "fn_function_keys": {
                "f1":  "vk_consumer_brightness_down",
                "f2":  "vk_consumer_brightness_up",
                "f3":  "f3",
                "f4":  "f4",
                "f5":  "vk_consumer_illumination_down",
                "f6":  "vk_consumer_illumination_up",
                "f7":  "vk_consumer_previous",
                "f8":  "vk_consumer_play",
                "f9":  "vk_consumer_next",
                "f10": "mute",
                "f11": "volume_down",
                "f12": "volume_up"
            }
        }
    ]
}
```

**Comment by nicklear** [comment]
No luck I'm afraid. I tried also changing f5 with "f5": "f5", in that line but that also didn't work

**Comment by andrewplummer** [comment]
This is working nearly perfectly for me, however reclaiming just F4 seems to turn ALL the media keys into F* keys. Here is my config:

```
{
    "profiles": [
        {
            "name": "Default profile",
            "selected": true,
            "simple_modifications": {
              "caps_lock":  "escape"
            },
            "fn_function_keys": {
                "f4":  "f4"
            }
        }
    ]
}
```

**Comment by jklp** [comment]
@andrewplummer I just tested with 0.90.47 and you're correct, if you just specify "f4" on it's own it converts all f* keys to function keys.  As a workaround you need to specify all the other keys (as @tekezo mentioned).

In saying that, once you specify the remaining f* keys, hitting fn-f* does call the f* keys and regular f* calls the virtual key (e.g. dim/brighten display)

**Comment by andrewplummer** [comment]
@jklp That seems to work now! ...could swear it didn't before, but anyway that's basically all I need! Thanks!

**Comment by incandescentman** [comment]
F3 and F4 aren't registering for me either, and when I do `Show Keyboard Viewer`, they don't register as keypresses. Mapping them to themselves like this didn't work either. 

     "fn_function_keys": {
                "f4":  "f4"
               }
I was able to successfully map all but F3 and F4. Mapping F8 to itself did work. Just not F3 and F4. When I mapped them back to Mission Control and Launchpad, they worked to do that again, but I can't get them to work for any other function.
